websites/B/Battlefield Portal/presence.ts

```diff
@@ -26,9 +26,11 @@ presence.on("UpdateData", async () => {
   const presenceData: PresenceData = {
       largeImageKey: "img1"
     },
-    [block, time, buttons, name, description]: boolean[] = await Promise.all(
-      ["block", "time", "buttons", "name", "desc"].map(presence.getSetting)
-    ),
+    block = await presence.getSetting<boolean>("block"),
+    time = await presence.getSetting<boolean>("time"),
+    buttons = await presence.getSetting<boolean>("buttons"),
+    name = await presence.getSetting<boolean>("name"),
+    description = await presence.getSetting<boolean>("desc"),
```

Comment:
Keep the Promise.all, just assign types like this
```suggestion
    [block, time, buttons, name, description] = await Promise.all([
			presence.getSetting<boolean>("block"),
    	presence.getSetting<boolean>("time"),
    	presence.getSetting<boolean>("buttons"),
    	presence.getSetting<boolean>("name"),
    	presence.getSetting<boolean>("desc")
]),
    
```